lib/God/Methods.js

```diff
@@ -221,11 +240,21 @@ module.exports = function(God) {
     if (!pid) return cb({msg : 'no pid passed or null'});
 
     try {
-      treekill(pid);
+      var p = God.findOneByPID(pid);
+
+      if(p === null || p.treekill === true) {
+        treekill(pid);
+      } else {
+        //if we don't want to kill the tree we just SIGINT the process
+        //sending SIGINT with the process group will kill all child processes but the detached ones
+        process.kill(-pid, 'SIGINT'); //- act as as process group
```

Comment:
Yes negative pid. See [this interesting answer](http://unix.stackexchange.com/questions/45426/why-would-ctrl-c-behave-differently-than-kill-2).